.gitignore

```diff
@@ -23,7 +23,7 @@ yarn-error.log
 *secrets*.yml
 !helm_deploy/check-my-diary/templates/secrets.yaml
 !helm_deploy/secrets-example.yaml
-test_results
+test-results
```

Comment:
this looks wrong - the directory was renamed as a result of copying the template configuration into this project.  You should delete the `test-results` directory locally instead as it won't now be recreated.